### PR TITLE
RIP 76 Email someone else page

### PR DESCRIPTION
**Pull request by reggieb**
Add functionality to allow secondary contact email to be gathered.

### CODE REVIEW

File: app/views/flood_risk_engine/enrollments/steps/_email_someone_else.html.erb [hunk 1]
```diff
@@ -1 +1,26 @@
...
+  <%= form_group_and_validation(form, :email_address) do -%>
+    <div class="form-group">
+      <%= f.label(:email_address, t(".email_address.label"), class: "form-label") %>
+      <%= f.email_field(:email_address, class: "form-control") %>
+    </div>
+  <% end %>
+
+  <%= form_group_and_validation(form, :email_address_confirmation) do -%>
+  <div class="form-group">
+    <%= f.label(
```
Comment (woodpigeon) [inline app/views/flood_risk_engine/enrollments/steps/_email_someone_else.html.erb]: indentation missing around `<div class="form-group">` :smile:

Comment (reggieb) [inline app/views/flood_risk_engine/enrollments/steps/_email_someone_else.html.erb]: Good spot

File: config/locales/flood_risk_engine/enrollments/steps/_email_someone_else.en.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            label: Email address for registration copy (optional)
+          email_address_confirmation:
+            label: Confirm email address if entered above
+          errors:
+            email_address:
+              blank: Please enter both the email and the confirmation
+              invalid: Enter a valid email address - there’s a mistake in that one
+            email_address_confirmation:
+              blank: Enter the email address again to confirm it
+              invalid: The email addresses you’ve entered don’t match
```
Comment (woodpigeon) [inline config/locales/flood_risk_engine/enrollments/steps/_email_someone_else.en.yml]: No newline at end of file. Is there a way to configure net beans to always add a new line (if missing) when the file is saved?

Comment (reggieb) [inline config/locales/flood_risk_engine/enrollments/steps/_email_someone_else.en.yml]: There's a setting to get rid of trailing white space, but not ensure a new line at the end of a file.

Isn't there a rubocop check for that?

**Comment by woodpigeon** [comment]
Happy for it to :ship: when issues resolved, Tom reviewed, squashed

**Comment by autotelik** [comment]
seems light on view/controller specs, other than that :ship:

**Comment by reggieb** [comment]
@autotelik I reviewed the existing controller specs when I started building the tests (hence me refactoring some of them by splitting them out into their own files - so the core step controller test doesn't get too big). My conclusion was that the controller tests were generic - testing that the right form was loaded for example and that was already adequately tested. So I concentrated on what's difference to this page - the form.

**Pull request merged**